src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -209,7 +209,7 @@ internal bool IsMLDataIngestionPipelineinBeta
         {
             get
             {
-                return DynamoModel.FeatureFlags?.CheckFeatureFlag("IsMLDataIngestionPipelineinBeta", false) ?? false;
+                return DynamoModel.FeatureFlags?.CheckFeatureFlag("IsMLDataIngestionPipelineinBeta", true) ?? true;
```

Comment:
@QilongTang Confirm if the default value for this flag should be true and in the case feature flags object is null.